org.nomacs.ImageLounge.json

```diff
@@ -0,0 +1,107 @@
+{
+    "app-id": "org.nomacs.ImageLounge",
+    "default-branch": "stable",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.14",
+    "sdk": "org.kde.Sdk",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--filesystem=home"
+    ],
+    "command": "nomacs",
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/aclocal",
+        "/share/man",
+        "*.la", "*.a"
+    ],
+    "build-options": {
```

Comment:
Why are you disabling debuginfo?

Reply:
I don't know : ) -> the config was written by @egrath . So it's best to remove `"*.la", "*.a"`

Reply:
As far as i understood, setting strip:true and no-debuginfo:true removes all Debug information from the executables, which was the desired behavior, as it's a release build.

Reply:
Debuginfo is automatically extracted into a separate extension:
https://docs.flatpak.org/en/latest/debugging.html#creating-a-debug-extension

Reply:
ok, so I'll set `"no-debuginfo": false,`

Reply:
You can just delete entire `build-options`, these are default settings.

Reply:
thanks!